benchmarker.py

```diff
@@ -63,6 +61,15 @@ def exit_handler() -> None:
     # also factorio.zip and maps.zip can be left over in rare cases and fail the reinstall.
 
 
+def get_factorio_version(factorio_bin: str, full: bool = False) -> str:
+    """returns the version string of the installed factorio instance"""
+    factorio_log_version = os.popen(f"{factorio_bin} --version").read()
+    result = factorio_log_version.splitlines()[0].split()[1]
+    if full:
+        result += " " + factorio_log_version.splitlines()[0].split()[5][:-1]
+    return result
```

Comment:
```suggestion
    if full:
        result += " " + factorio_log_version.splitlines()[0].split()[4][:-1]
        result += " " + factorio_log_version.splitlines()[0].split()[5][:-1]
    return result
```
**1.1.72 win64 full** - there is the name of the operating system here
I haven't checked on Linux

Reply:
yes, it's the same on linux, but I thought it was `...headless, full)` for the headless version. but it turned out to be `linux64, headless` which makes more sense.